include/xtensor-io/xhighfive.hpp

```diff
@@ -25,6 +25,18 @@
 
 namespace xt
 {
+    namespace detail
+    {
+        inline void error(const HighFive::File& file, const std::string &path, std::string message)
```

Comment:
looks good! minor formatting: ampersand at path, and spaces around `+` signs :)

`const std::string& path`

`"path: '" + path ...`

Reply:
Got it!